packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.test.tsx

```diff
@@ -680,11 +694,17 @@ describe('ComboBox', () => {
     inputElement.simulate('blur');
     _verifyStateVariables(componentRef, false, [...DEFAULT_OPTIONS, { ...comboBoxOption }], [3]);
 
-    // this should toggle the checkbox
     inputElement.simulate('focus');
     _verifyStateVariables(componentRef, true, [...DEFAULT_OPTIONS, { ...comboBoxOption }], [3]);
     inputElement.simulate('input', { target: { value: 'ManuallyEnteredValue' } });
     _verifyStateVariables(componentRef, true, [...DEFAULT_OPTIONS, { ...comboBoxOption }], [3]);
+
+    // this should toggle the checkbox
+    const buttonElement: any = wrapper.find('.ms-ComboBox button')! as any;

```

Comment:
Do you really need both `!` and `any`? That seems redundant

Reply:
Good call, I'll clean this up.